### PR TITLE
[Fix] Slippage in bridges and swaps

### PR DESCRIPTION
**Pull request by leablock**
# Name of the feature/issue

Fixes related to the "slippage" field in bridges

## Description
- Don't send any slippage in the first request generating the quote and then use the slippage from its response in the next calls as a default value. 
  - In case it doesn't exist use 3% because it's sometimes used by LiFi
- Fix reset button
- Fix reset button
- Change front-run warning 
- Allow 0% slippage in advanced settings

### CODE REVIEW

**Review by fpezzola — APPROVED** [review]
LGTM! Please check my comment

File: packages/ui/src/routes/bridge/BridgeConfirmPage.tsx [hunk 1]
```diff
@@ -435,19 +441,17 @@ const BridgeConfirmPage: FunctionComponent<{}> = () => {
...
             await fetchQuoteParams()
             timeoutRef = setTimeout(fetchParams, QUOTE_REFRESH_TIMEOUT)
         }
 
         fetchParams()
-
         // Cleanup timer
         return () => {
             isValidFetch = false
-            timeoutRef && clearTimeout(timeoutRef)
```
[the remark this answers is not in the document]
Comment (leablock) [reply packages/ui/src/routes/bridge/BridgeConfirmPage.tsx:450]: clearTimeout accepts `NodeJS.Timeout | undefined`

[the remark this answers is not in the document]
Comment (leablock) [reply packages/ui/src/routes/bridge/BridgeConfirmPage.tsx:450]: @fpezzola

[the remark this answers is not in the document]
Comment (fpezzola) [reply packages/ui/src/routes/bridge/BridgeConfirmPage.tsx:450]: But the type of `timeoutRef` is `NodeJS.Timeout | null`

[the remark this answers is not in the document]
Comment (fpezzola) [reply packages/ui/src/routes/bridge/BridgeConfirmPage.tsx:450]: Really good catch. But I'd update the `timeoutRef`

**Pull request merged**